cpmpy/solvers/exact.py

```diff
@@ -94,22 +93,13 @@ def __init__(self, cpm_model=None, subsolver=None):
         from exact import Exact as xct
 
         # initialise the native solver object
-        self.xct_solver = xct()
-        self.xct_solver.setOption("inp-purelits", "0")    # no dominance breaking to preserve solutions
-        self.xct_solver.setOption("inp-dombreaklim", "0") # no dominance breaking to preserve solutions
+        options = list(kwargs.items()) # options is a list of string-pairs, e.g. [("verbosity","1")]
+        options = [(opt[0], str(opt[1])) for opt in options] # Ensure values are also strings
```

Comment:
The "-" as separator will cause issues. The "options" come from optional keyword arguments via "kwargs", which does not allow for "-" inside the names of the method arguments. As an alternative, I tried requiring the arguments to be supplied to cpmpy with a "_" as seperator and then transforming them back using:
```python 
options = [(opt[0].replace("_", "-"), str(opt[1])) for opt in options] # Python does not support "-" in parameter names
```
But then we need to add a disclaimer to the documentation (and not just solely refer to Exact's list of arguments)

Reply:
That should be fine - for other solvers this happens as well, but you can do the following hack:
```python
solver = CPM_Exact(**{"cool-argument-with-dashes" : 42})
```

Reply:
I see you added some documentation for it.